### ISSUE TITLE
Ambiguous information about form fields being required or not

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1522]
**Describe the bug**
When browsing the Basic Info page with a screen reader (VoiceOver) some of the textboxes that are not indicated to be required on the screen are said to be required by the screen reader.

**To Reproduce**
1. Turn on the screen reader (Voice Over)
2. Go to 'https://www.topcoder-dev.com/settings/profile'
3. Click on 'Basic Info'
4. Navigate to the 'Address' field for instance
5. See error.

**Expected behavior**
The information about textboxes should be the same both on the screen and in the screen reader voice. 

**Screenshots** Now adress is required or not?
<img width="1280" alt="screenshot 2018-09-16 21 37 16" src="https://user-images.githubusercontent.com/35376779/45600351-99056e80-b9fb-11e8-8ec5-6e3dcbc4016c.png">

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Safari
 - Version: 11.1.2

@ThomasKranitsas
